src/gui/dialogs/addon/manager.cpp

```diff
@@ -738,6 +739,21 @@ void addon_manager::publish_addon(const addon_info& addon)
 		}
 	}
 
+	// the passphrase isn't provided, prompt for it
+	if(cfg["passphrase"].str().empty()) {
```

Comment:
Why not just `check `cfg["passphrase"].empty()`?

Reply:
No idea, will fix.